### PR TITLE
docs(tabs): Adds css documentation for tabs component

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

File: next.clayui.com/content/docs/css/components/tabs.md [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+        Skinny extraction, id trifecta qui trifecta grinder. Barista robusta arabica breve ut skinny milk beans macchiato carajillo espresso. Cultivar single shot brewed, coffee steamed to go wings to go cortado. Grinder, siphon coffee acerbic espresso cinnamon crema breve.
+    </div>
+    <div aria-labelledby="buttonSupportedClientsTab" class="fade tab-pane" id="buttonSupportedClients" role="tabpanel">
+        At americano, viennese variety iced grounds, grinder froth and pumpkin spice aromatic. Cultivar aged lungo, grounds café au lait, skinny, blue mountain, in variety sugar shop roast. Wings, blue mountain affogato organic cappuccino java plunger pot. Single shot variety pumpkin spice seasonal skinny barista carajillo robust cream.
+    </div>
+</div>
+```
+
+## Justified
+
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/tabs.md]: We should probably add a description here and what class name is needed

File: next.clayui.com/content/docs/css/components/tabs.md [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+description: "Tabs organize similar content together into individual sections in the same page."
+---
+
+<div class="nav-toc">
+
+- [Classic](#classic)
+- [Modern](#modern)
+- [Variations](#variations)
+    - [Buttons](#buttons)
+	- [Justified](#justified)
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/tabs.md]: Should this be on the same line as buttons? We can use justified without buttons right?

Comment (diegonvs) [reply next.clayui.com/content/docs/css/components/tabs.md]: Same related issue of ghost spaces. Fixing!

File: next.clayui.com/content/docs/css/components/tabs.md (hunk 1 above)
Comment (diegonvs) [reply next.clayui.com/content/docs/css/components/tabs.md]: Yep. Adding docs for this!

**Pull request merged**